src/TraceEvent/Symbols/SymbolReader.cs

```diff
@@ -1072,7 +1072,7 @@ internal bool GetPhysicalFileFromServer(string serverPath, string pdbIndexPath,
                         {
                             m_log.WriteLine("FindSymbolFilePath: In task, sending HTTP request {0}", fullUri);
 
-                            var responseTask = HttpClient.GetAsync(fullUri);
+                            var responseTask = HttpClient.GetAsync(fullUri, HttpCompletionOption.ResponseHeadersRead);
```

Comment:
I'm not against it, but I'm curious what problem ResponseHeaerRead solves.

Reply:
This ensures that the call to `HttpClient.GetAsync` returns quickly and allows the code in `SymbolReader` to show status for the download.  If we don't do this, then symbol servers that download the entire blob (e.g. new symweb, msdl, azdo, etc.) will not show any download status in the logs until the call to `HttpClient.GetAsync` completes.  For large files, this could cause a timeout.  This is especially true for `msedge` PDBs which are greater than 1GB in size.

The reason that you may not have noticed this (as I didn't) is that the existing symweb just downloads a redirect to a UNC path, and so the download from the symbol server is tiny and super quick.  Then we go copy the file from the UNC path.  When this happens, it's a 2 step operation rather than for a blob-based symbol server, where the file comes in the response of the original request.

Reply:
Thanks for the explanation. Makes sense to me.